### ISSUE TITLE
GraalVM jlink fails with Error: Hash of jdk.crypto.ec on MacOS and Windows

### ISSUE THREAD

**Issue by renatoathaydes** [oracle/graal#4047]
I am trying to package a modular Java application with `jlink`.

It works fine on OpenJDK in all operating systems, but as I am using GraalVM.js, I wanted to compile and package it using GraalVM.

However, the build seems to only succeed on Linux. On both MacOS and Windows, I am getting the following error:

```
FAILURE: Build failed with an exception.
Error: Hash of jdk.crypto.ec (9fdafad8d06ba435ce23c35cc5fd0a42984bbfb72e4a50795ba72780159a3161) differs to expected hash (9ef195cf81e2d4fd19501994c15d4860f202cae5fd896ffcc1878b7a84d46754) recorded in java.base
```

* [Linux build](https://github.com/renatoathaydes/rawhttp/runs/4304958401?check_suite_focus=true) (OK)
* [MacOS build](https://github.com/renatoathaydes/rawhttp/runs/4304958457?check_suite_focus=true) (FAIL)
* [Windows build](https://github.com/renatoathaydes/rawhttp/runs/4304958499?check_suite_focus=true) (FAIL)

It also fails on my Mac laptop.

**Steps to reproduce the issue**

1. `git clone --depth 1 git@github.com:renatoathaydes/rawhttp.git`
2. `git checkout java-modularization-graalvm`
3. `./gradlew :rawhttp-cli:jlink`

**Describe GraalVM and your environment:**

 ```
▶ java --version                                      
openjdk 11.0.12 2021-07-20
OpenJDK Runtime Environment GraalVM CE 21.2.0 (build 11.0.12+6-jvmci-21.2-b08)
OpenJDK 64-Bit Server VM GraalVM CE 21.2.0 (build 11.0.12+6-jvmci-21.2-b08, mixed mode, sharing)
```

**Comment by renatoathaydes** [comment]
I think that to reproduce, trying to jlink a simple modular Java project with at least these modules should suffice:

`--add-modules', 'java.base,jdk.crypto.ec`

The crypto module is required for HTTPS to work.

**Comment by rodrigar-mx** [comment]
Hi @renatoathaydes. Could you please try using the latest graalvm ce 22.0.0-dev build (https://github.com/graalvm/graalvm-ce-dev-builds/releases) and share your results. The error should be already fixed.

**Comment by rodrigar-mx** [comment]
This should be fixed in the upcoming January releases (22.0.0, 21.3.1).
